### PR TITLE
add timestamp generate-roblox-std (issue 82)

### LINKED ISSUE #82: Timestamp generate-roblox-std creation
Currently, selene gives no information about when `roblox.std` was generated. It would be nice to have selene give some insight as to when it was generated with a timestamp.

Current information:
```toml
# This file was @generated by generate-roblox-std
```

A better insight might look like
```toml
# This file was @generated by generate-roblox-std at 23-01-20 7:39pm
```

### PR DESCRIPTION
**Pull request by 05st**
Closes #82

### CODE REVIEW

File: selene/src/roblox.rs [hunk 1]
```diff
@@ -1,3 +1,6 @@
+extern crate chrono;
```
Comment (Kampfkarren) [inline selene/src/roblox.rs]: extern crate isn't necessary in modern Rust

File: selene/src/roblox.rs [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+use chrono::Local;
+
```
Comment (Kampfkarren) [inline selene/src/roblox.rs]: Can you remove the extra whitespace here? cargo fmt should fix this.

File: selene/Cargo.toml [hunk 3]
```diff
@@ -27,6 +27,7 @@ structopt = "0.3"
 termcolor = "1.0"
 threadpool = "1.7"
 toml = "0.5"
+chrono = "0.4"
```
Comment (Kampfkarren) [inline selene/Cargo.toml]: Dependencies should be in alphabetical order

**Comment by 05st** [comment]
fixed, sorry

**Comment by Kampfkarren** [comment]
Looks good, I'll test it when I can.

**Comment by Kampfkarren** [comment]
Works on my machine!

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## Unreleased
 
 ### Added
+- Added timestamp to generate-roblox-std
```
Comment (Kampfkarren) [inline CHANGELOG.md]:
```suggestion
- Added timestamp to generated Roblox standard library
```

This is used outside of generate-roblox-std itself, so this is me clear. Last critique, I promise.

**Comment by Kampfkarren** [comment]
Thanks! Waiting on CD now.

**Pull request merged**